ql/src/java/org/apache/hadoop/hive/ql/Context.java

```diff
@@ -90,6 +91,7 @@ public class Context {
   String originalTracker = null;
   private CompilationOpContext opContext;
   private final Map<String, ContentSummary> pathToCS = new ConcurrentHashMap<String, ContentSummary>();
+  public static final String REWRITE_POLICY = "hive.rewrite.policy";
```

Comment:
please move it to `InputFormatConfig.class`: REWRITE_POLICY="hive.rewrite.data.policy";

Reply:
done